dir.props

```diff
@@ -26,6 +26,7 @@
     <XunitVersion>2.1.0</XunitVersion>
     <CompilerToolsVersion>1.2.0-beta1-20160108-01</CompilerToolsVersion>
     <MicroBuildVersion>0.2.0</MicroBuildVersion>
+    <GitVersioningVersion>1.4.30</GitVersioningVersion>
```

Comment:
This file is interesting. @AndyGerlicher can you sanity check? It conflicted with the x86/x64 build and with the GitVersioning bump.